### PR TITLE
implement BookCard component

### PR DESCRIPTION
**Pull request by tranchinh123**

### CODE REVIEW

File: bookshelter-app/src/components/BookCard/index.module.css [hunk 1]
```diff
@@ -4,3 +4,52 @@
   border-radius: 5px;
   background-color: var(--background-color-primary);
 }
+.card {
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  width: 282px;
+  height: 369px;
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/BookCard/index.module.css]: why fixed width/height?

**Comment by huypham-agilityio** [comment]
@tranchinh123 css only? How about react component

File: bookshelter-app/src/components/BookCard/index.module.css (hunk 1 above)
Comment (tranchinh123) [reply bookshelter-app/src/components/BookCard/index.module.css]: Fixed

File: bookshelter-app/src/components/BookCard/index.tsx [hunk 2]
```diff
@@ -1,15 +1,33 @@
 import styles from './index.module.css';
-const BookCard = () => {
+interface BookCardProps {
+  name: string;
+  author: string;
+  published: number;
+  url: string;
+  handleClick: () => void;
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/BookCard/index.tsx]: Use `on` prefix for prop names

File: bookshelter-app/src/components/BookCard/index.tsx [hunk 3]
```diff
@@ -1,15 +1,33 @@
...
+}
+
+const BookCard = ({
+  name,
+  author,
+  published,
+  url,
+  handleClick,
+}: BookCardProps) => {
+  const handleCardClick = () => {
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/BookCard/index.tsx]: no need define it

Comment (tranchinh123) [reply bookshelter-app/src/components/BookCard/index.tsx]: Fixed

File: bookshelter-app/src/components/BookCard/index.tsx (hunk 2 above)
Comment (tranchinh123) [reply bookshelter-app/src/components/BookCard/index.tsx]: Fixed

**Pull request merged**